docs/specification/external-terms.md

```diff
@@ -20,3 +20,32 @@ http://xmlns.com/foaf/spec/
 https://www.w3.org/TR/vocab-org/
 
 * [`org:Organization`](https://www.w3.org/TR/vocab-org/#org:Organization)
+
+## QUDT
+
+http://qudt.org/doc/2016/DOC_SCHEMA-QUDT-v2.0.html
+
+* [`qudt:QuantityValue`](http://qudt.org/doc/2016/DOC_SCHEMA-QUDT-v2.0.html#Classes)
+* [`qudt:unit`](http://qudt.org/doc/2016/DOC_SCHEMA-QUDT-v2.0.html#Properties)
+* [`qudt:numericValue`](http://qudt.org/doc/2016/DOC_SCHEMA-QUDT-v2.0.html#Properties)
+
+### UNIT
+
+http://qudt.org/doc/2017/DOC_VOCAB-UNITS-BASE.html
+
+* [`unit:Number`](http://qudt.org/doc/2017/DOC_VOCAB-UNITS-BASE.html#Instances)
```

Comment:
Not clear why you are calling out this particular unit?  Even if true that we use it a lot.

Reply:
Since we use it a lot people should know when to find it.